web/modules/custom/pineapple_general/pineapple_general.module

```diff
@@ -44,6 +44,23 @@ function pineapple_general_field_widget_form_alter(&$element, \Drupal\Core\Form\
       }
     }
   }
+
+  if (key_exists('form_id', $form_state->getBuildInfo())) {
+    $build_info = $form_state->getBuildInfo();
+    if ($build_info['form_id'] == 'node_article_edit_form' || 'node_recipe_edit_form') {
```

Comment:
This should be `if ($build_info['form_id'] == 'node_article_edit_form' || $build_info['form_id'] == 'node_recipe_edit_form')` and also the code will not run for newly created articles or recipes